packages/integration-tests/tests/bridge_erc20.ts

```diff
@@ -141,8 +196,76 @@ describe('Bridging ERC20 tokens', async function () {
       await env.eth.fuelERC20Gateway.getAddress()
     ).toLowerCase();
 
-    eth_testToken = await getOrDeployECR20Contract(env);
-    eth_testTokenAddress = (await eth_testToken.getAddress()).toLowerCase();
+    let usdt_testToken = await getOrDeployCustomTokenContract(env);
+    let usdc_testToken = await getOrDeployCustomTokenContract(env);
+    let wbtc_testToken = await getOrDeployCustomTokenContract(env);
+    weth_testToken = await getOrDeployCustomWETHContract(env);
+
+    // Get the runtime bytecode.
+    const runtimeBytecodeUSDTToken: string = await env.eth.provider.getCode(
+      await usdt_testToken.getAddress()
+    );
+    const runtimeBytecodeUSDCToken: string = await env.eth.provider.getCode(
+      await usdc_testToken.getAddress()
+    );
+    const runtimeBytecodeWBTCToken: string = await env.eth.provider.getCode(
+      await wbtc_testToken.getAddress()
+    );
+    const runtimeBytecodeWETHToken: string = await env.eth.provider.getCode(
+      await weth_testToken.getAddress()
+    );
+
+    // set code for mainnet addresses
+    await env.eth.provider.send('hardhat_setCode', [
+      USDT_ADDRESS,
+      runtimeBytecodeUSDTToken,
+    ]);
+    await env.eth.provider.send('hardhat_setCode', [
+      USDC_ADDRESS,
+      runtimeBytecodeUSDCToken,
+    ]);
+    await env.eth.provider.send('hardhat_setCode', [
+      WBTC_ADDRESS,
+      runtimeBytecodeWBTCToken,
+    ]);
+    await env.eth.provider.send('hardhat_setCode', [
```

Comment:
Move this to hardhat deploy scripts at `packages/solidity-contracts/deploy/hardhat`, we want this to be available to anyone that runs the envs

Reply:
if you add it to hardhat deploy scripts, they should be available in e2e tests

Reply:
We need them in hardhat scripts because other repositories will be using the e2e environment - ie, someone will run our docker images and expect everything to already be deployed and ready.

Why do you need external inputs? afaik, all the information we need to deploy these tokens is static. We know how USDC, USDT and WETH behave.